### DISCUSSION TITLE
Tackling Monopolies

### DISCUSSION THREAD

**Discussion by giscus** [Andrew-Chen-Wang/blog#3] [General]
# Tackling Monopolies

Monopolies resist change, maintaining the status quo of today's level of inadequacy

https://blog.andrewcwang.com/blog/2024-08-15-solving-efficiency-against-monopolies/

**Comment by Andrew-Chen-Wang** [comment]
Was just thinking about the economist's perspective on markets: markets derive profits from inefficiency; one may conclude software isn't unique in the sense of what they're solving. Automobiles solve the transportation issue... or did it actually reinvent transportation to exclude the need to take care of horses?

I think it's best to look at monopolies from a businessman's perspective. Given an industry, what at its core (or on a per segment basis if targeting a niche) is the most desirable state? Softer plushies? A B2B SaaS perfectly designed for all companies to most efficiently do things? So obviously, it's not all about inefficiencies but rather inherent problems businesses are looking to solve.
